packages/commercetools/composables/src/useCheckout/initFields.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  personalDetails,
+  shippingDetails,
+  billingDetails,
+  initialDetails,
+  isShippingAddressCompleted,
+  isBillingAddressCompleted,
+  isPersonalDetailsCompleted,
+  chosenShippingMethod
+} from './shared';
+
+const initFields = (cart) => {
+  personalDetails.value.email = cart.customerEmail;
+  shippingDetails.value = { ...initialDetails, ...shippingDetails.value, ...cart.shippingAddress };
+  billingDetails.value = cart.billingAddress || initialDetails;
```

Comment:
Why do we make a copy of `cart.shippingAddress` (so it won't be mutated) but `cart.billingAddress` is just passed to the export here so it can be mutated by consumers of this module?

Reply:
it's easier to have local variables related to the checkout, instead of operating on the cart directly. We'll move some  of the functionalities from here to core,  so these fields will be easy to move